scripts/main.js

```diff
@@ -71,4 +71,57 @@ const departments = {
     }
 }
 
-console.log(departments);
\ No newline at end of file
+console.log(departments);
+console.log("De afdeling sales heeft " + departments.sales.numberOfEmployees + " medewerkers")
+console.log(`De afdeling sales heeft " ${departments.sales.numberOfEmployees}  " medewerkers`)
+console.log("Marketing is een leuke afdeling om te werken. " + departments.marketing.description)
+console.log("De afdeling Customer Service heeft " + departments["customer-service"].numberOfEmployees + " medewerkers")
+console.log("Sales is een uitdagende afdeling om te werken als Verkoopmanager. " + departments.sales.jobs[1].description)
+
+const name = prompt('Hoi! Hoe heet je?');
+console.log(name);
+
+////////// Opdracht 2 //////////
+
+const userInput = prompt('Over welke afdeling wil je meer informatie? Kies uit: [marketing / sales / customer-service]').toLowerCase();
+
+if (userInput in departments) {
+    console.log(`Je koos ${userInput}. ${departments[userInput].description}`);
```

Comment:
${userInput} is een getal, dus het gekozen nummer wordt in de console gelogd in plaats van de functietitel.